operator/controllers/suite_test.go

```diff
@@ -1,81 +1,80 @@
-// http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
-
-var cfg *rest.Config
-var k8sClient client.Client
-var testEnv *envtest.Environment
-
-func TestAPIs(t *testing.T) {
-	RegisterFailHandler(Fail)
-
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
-}
-
-var _ = BeforeSuite(func(done Done) {
-	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
-
-	By("bootstrapping test environment")
-	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join("..", "config", "crd", "bases")},
-	}
-
-	var err error
-	cfg, err = testEnv.Start()
-	Expect(err).ToNot(HaveOccurred())
-	Expect(cfg).ToNot(BeNil())
-
-	err = httpv1alpha1.AddToScheme(scheme.Scheme)
-	Expect(err).NotTo(HaveOccurred())
-
-	// +kubebuilder:scaffold:scheme
-
-	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
-	Expect(err).ToNot(HaveOccurred())
-	Expect(k8sClient).ToNot(BeNil())
-
-	close(done)
-}, 60)
-
-var _ = AfterSuite(func() {
-	By("tearing down the test environment")
-	err := testEnv.Stop()
-	Expect(err).ToNot(HaveOccurred())
-})
+// import (
+// 	"path/filepath"
+// 	"testing"
+
+// 	. "github.com/onsi/ginkgo"
+// 	. "github.com/onsi/gomega"
+// 	"k8s.io/client-go/kubernetes/scheme"
+// 	"k8s.io/client-go/rest"
+// 	"sigs.k8s.io/controller-runtime/pkg/client"
+// 	"sigs.k8s.io/controller-runtime/pkg/envtest"
+// 	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
+// 	logf "sigs.k8s.io/controller-runtime/pkg/log"
+// 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+// 	httpv1alpha1 "github.com/kedacore/http-add-on/operator/api/v1alpha1"
+// 	// +kubebuilder:scaffold:imports
+// )
+
+// // These tests use Ginkgo (BDD-style Go testing framework). Refer to
+// // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
+
+// var cfg *rest.Config
+// var k8sClient client.Client
+// var testEnv *envtest.Environment
+
+// func TestAPIs(t *testing.T) {
+// 	RegisterFailHandler(Fail)
+
+// 	RunSpecsWithDefaultAndCustomReporters(t,
+// 		"Controller Suite",
+// 		[]Reporter{printer.NewlineReporter{}})
+// }
+
+// var _ = BeforeSuite(func(done Done) {
+// 	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
+
+// 	By("bootstrapping test environment")
+// 	testEnv = &envtest.Environment{
+// 		CRDDirectoryPaths: []string{filepath.Join("..", "config", "crd", "bases")},
+// 	}
+
+// 	var err error
+// 	cfg, err = testEnv.Start()
+// 	Expect(err).ToNot(HaveOccurred())
+// 	Expect(cfg).ToNot(BeNil())
+
+// 	err = httpv1alpha1.AddToScheme(scheme.Scheme)
+// 	Expect(err).NotTo(HaveOccurred())
+
+// 	// +kubebuilder:scaffold:scheme
+
+// 	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
+// 	Expect(err).ToNot(HaveOccurred())
+// 	Expect(k8sClient).ToNot(BeNil())
+
+// 	close(done)
+// }, 60)
+
+// var _ = AfterSuite(func() {
+// 	By("tearing down the test environment")
+// 	err := testEnv.Stop()
+// 	Expect(err).ToNot(HaveOccurred())
+// })
```

Comment:
@khaosdoctor can you delete or uncomment this file? #43 will add & update it

Reply:
I'll delete it hete

Reply:
ah! I'm sorry for the completely contradictory requests here and https://github.com/kedacore/http-add-on/pull/47/files#r581350008. Let's keep this as you have it right now and I'll take care of it i #43